spec/forms/idv/profile_form_spec.rb

```diff
@@ -59,7 +60,8 @@
   describe 'ssn uniqueness' do
     context 'when ssn is already taken by another profile' do
       it 'is invalid' do
-        create(:profile, ssn: '1234', user_id: 456)
+        diff_user = create(:user)
```

Comment:
Although the stubbing might be more tricky here. Not a big deal if it's too complicated.